spyder_kernels/console/kernel.py

```diff
@@ -759,27 +763,35 @@ def set_special_kernel(self, special):
         raise NotImplementedError(f"{special}")
 
     @comm_handler
-    def update_syspath(self, path_dict, new_path_dict):
+    def update_syspath(self, new_path, prioritize):
         """
         Update the PYTHONPATH of the kernel.
 
-        `path_dict` and `new_path_dict` have the paths as keys and the state
-        as values. The state is `True` for active and `False` for inactive.
+        `new_path` corresponds to the new state of the PYTHONPATH.
+        `prioritize` determines whether to prioritize PYTHONPATH in sys.path.
 
-        `path_dict` corresponds to the previous state of the PYTHONPATH.
-        `new_path_dict` corresponds to the new state of the PYTHONPATH.
+        A copy of sys.path is made at instantiation, which should be clean,
+        so we can just prepend/append to the copy without having to explicitly
+        remove old user paths. PYTHONPATH can just be overwritten.
```

Comment:
Please rewrite this docstring in the Numpy docstring style, i.e. with Parameters and Notes sections.